internal/cli/atlas/search/nodes/list.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 MongoDB Inc
+// Copyright 2024 MongoDB Inc
```

Comment:
in Amazon the year in copyright was "from" when it was created the file, it was not needed to changed it, don't know here, i would assume there is no need either

Reply:
The file was created this week. I copied the copyright from somewhere else. That's why it's updated to 2024